code/game/objects/structures/crates_lockers/crates/gear_loadout.dm

```diff
@@ -1052,37 +1052,108 @@
 	new /obj/item/device/modkit/multi_species(src)
 
 /obj/structure/closet/crate/secure/gear_loadout/exclusionist/fill()
-	new /obj/item/clothing/under/gearharness(src)
-	new /obj/item/clothing/under/gearharness(src)
-	new /obj/item/clothing/under/gearharness(src)
-	new /obj/item/clothing/under/gearharness(src)
-	new /obj/item/clothing/under/gearharness(src)
-	new /obj/item/clothing/suit/armor/exclusionist(src)
-	new /obj/item/clothing/suit/armor/exclusionist(src)
-	new /obj/item/clothing/suit/armor/exclusionist(src)
-	new /obj/item/clothing/suit/armor/exclusionist(src)
-	new /obj/item/clothing/suit/armor/exclusionist(src)
-	new /obj/item/clothing/mask/exclusionist(src)
-	new /obj/item/clothing/mask/exclusionist(src)
-	new /obj/item/clothing/mask/exclusionist(src)
-	new /obj/item/clothing/mask/exclusionist(src)
-	new /obj/item/clothing/mask/exclusionist(src)
-	new /obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
-	new /obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
-	new /obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
-	new /obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
-	new /obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
-	new /obj/item/melee/chainsword(src)
-	new /obj/item/melee/chainsword(src)
-	new /obj/item/melee/chainsword(src)
-	new /obj/item/melee/chainsword(src)
+	new	/obj/item/clothing/under/gearharness(src)
+	new	/obj/item/clothing/under/gearharness(src)
+	new	/obj/item/clothing/under/gearharness(src)
+	new	/obj/item/clothing/under/gearharness(src)
+	new	/obj/item/clothing/under/gearharness(src)
+	new	/obj/item/clothing/suit/armor/exclusionist(src)
+	new	/obj/item/clothing/suit/armor/exclusionist(src)
+	new	/obj/item/clothing/suit/armor/exclusionist(src)
+	new	/obj/item/clothing/suit/armor/exclusionist(src)
+	new	/obj/item/clothing/suit/armor/exclusionist(src)
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/melee/chainsword(src)
+	new	/obj/item/melee/chainsword(src)
+	new	/obj/item/melee/chainsword(src)
+	new	/obj/item/melee/chainsword(src)
```

Comment:
Single space only

Reply:
No idea why this is all changed, this isn't code I've touched. Or uhhh I didn't think it was